examples/trusted_database/proto/trusted_database_command.proto

```diff
@@ -23,5 +23,5 @@ import "oak_services/proto/grpc_invocation.proto";
 
 message TrustedDatabaseCommand {
   oak.invocation.GrpcInvocationReceiver invocation_receiver = 1;
-  repeated oak.examples.trusted_database.PointOfInterest points_of_interest = 2;
+  map<string, oak.examples.trusted_database.PointOfInterest> points_of_interest = 2;
```

Comment:
What is the key?

Would it make sense to just define this type once and for all in a proto, document it there, and use it everywhere else, instead of having so many (undocumented) copies?